sample/src/main/AndroidManifest.xml

```diff
@@ -14,7 +14,8 @@
         android:allowBackup="false">
 
         <activity android:name="no.finntech.rxactivityresponse.sample.SerializedRxSampleActivity"
-                  android:label="Sample">
+                  android:label="Sample"
```

Comment:
Navnet "Sample" gjør den vanskelig å finne i apps-lista når man skal skru av permissions for å teste